Client/Frontend/Browser/BrowserViewController/BrowserViewController+CookieNotificationBlocking.swift

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import SwiftUI
+import BraveUI
+import BraveShared
+import Data
+
+extension BrowserViewController {
+  func presentCookieNotificationBlockingCalloutIfNeeded() {
+    // Don't show this if we already enabled the setting
+    guard !FilterListResourceDownloader.shared.isEnabled(filterListUUID: FilterListResourceDownloader.cookieConsentNoticesUUID) else { return }
+    // Don't show this if we are presenting another popup already
+    guard !isOnboardingOrFullScreenCalloutPresented else { return }
+    // We only show the popup on second launch
+    guard !Preferences.General.isFirstLaunch.value else { return }
+    
+    #if DEBUG
+    // We reset this value on debug. If you don't want this popup to appear on debug, just enable the setting and it won't appear
+    FullScreenCalloutManager.enable(for: .blockCookieConsentNotices)
```

Comment:
Why do we reset this preference on debug? This will show all the time and like you mentioned we have to go toggle that setting to prevent this.

It might be better to use one of the `DebugFlag` in Preferences like we do in all the callouts and pop-ups or create a new skip flag and use ti same way.

Reply:
Yah i'll change this or remove it. It was just easier for testing